doc/method/method-sources.md

```diff
@@ -1,136 +1,65 @@
+are saved into a specific GitHub repository, and these records are then
+linked from the OTT taxonomy files and user interfaces so that
+provenance is always available.
+
+**Separation taxa**  
+This is a small curated tree containing 27 major groups such
+as animals, plants, and fungi.  Its purpose is to assist
+in separating homonyms.  If a node
+is found in one of these separation groups, then it will not match a
+node in a disjoint separation group, even if the name matches.
+
+**ARB-SILVA taxonomy processing**  
+The terminal taxa in the SILVA taxonomy are algorithmically-generated
+clusters of RNA sequences derived from GenBank records.  Rather than
+incorporate these idiosyncratic, fine-grained groupings into OTT, we
+use sequence record metadata to group the clusters into larger groups
+corresponding to NCBI taxa, and include those larger groups in OTT.
+
+We excluded SILVA's plant, animal, and fungal branches from OTT
+because these groups are well covered by other sources and poorly
+represented in SILVA.  For example, SILVA has only 299 taxa in
+Metazoa, compared with over 500,000 taxa under Metazoa in NCBI Taxonomy.
+
+**Extinct / extant annotations**  
 Curators requested information about whether taxa were extinct
 vs. extant.  (See below for the reason this was so important.) This
-was not present in any of our other sources, so we imported IRMNG,
+information was not explicitly present in any of our other sources, so we imported IRMNG,
 which logs the extinct / extant status of taxa.
-[reference]
-
-
-**Curation**  
-It is not uncommon to have taxa as OTUs in phylogenetic studies that do not
-occur in OTT.  This can happen when OTT does not contain the latest version of a
-source taxonomy, or when none of the source taxonomies contain the taxon. To
-handle this situation, we developed a user interface that allows curators to
-create new taxon records along with relevant documentation (publications,
-databases, and so on).  New taxon records are saved into a specific GitHub
-repository, and these records are then linked from the OTT taxonomy files and
-user interfaces so that provenance is always available.
-
-
-### Characterization of sources
-
-[NMF asks: What do the source taxonomies look like ("sampling", in
-terms of depth (# of levels, breadth (overlap? are they
-"comprehensive?"), homogeneity)?]
 
-[organizational note: this is not for the results section, but an
-assessment of the inputs to the method.  Some of this information
-could go in the table.]
+As a secondary heuristic, records from GBIF that originate from
+PaleoDB, and do not come from any other taxonomic source, are
+annotated extinct.  This is not completely reliable, as some PaleoDB taxa are extant.
+
+**Suppressed records**  
+Not all source taxonomy records are used.  The following are ignored:
+
+* GBIF backbone records that originate from IRMNG (IRMNG is imported separately)
+* GBIF backbone records that originate from IPNI
```

Comment:
Because Stephen said so, way back when, and I don't like to change curator decisions. I think he said IPNI was full of 'bad names' or something like that. Shall I ask him why? Or we could just say 'curator request'.

The IRMNG case could go either way. I think he had suppressed IRMNG because there were so many homonyms (by design!), but I'm not sure. We could include it now that our homonym handling is better, and now that GBIF's import of IRMNG is newer than ours.